metadata.json

```diff
@@ -29,15 +29,13 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8",
-        "9",
-        "10"
+        "10",
```

Comment:
I think the metatdata.json update should happen in a dedicated PR. there is no reason to include it here?

Reply:
These are #313 and #314 :wink:  …